Config/ProjectApollo/LEMSystems.cfg

```diff
@@ -119,73 +129,85 @@
 		SUITCIRCUITHEATEXCHANGERHEATING 1 2 1.0
 	</VALVE>
 	
-	<TANK> CDRSUIT <-0.5 0.5 0.0> 75.0 0.0001
+	<TANK> CDRSUIT <-0.5 0.5 0.0> 30.0 0.001
 		CHM    0 0.083125976 0.083042850 35.42964094 # Calculated for Suit Volume at 0.01PSI and 0F
 		#CHM    0 27.413288737 27.385875449 13463.26356 # Calculated for Suit Volume at 3.8PSI and 70F
         VALVE IN   1 1.0								# from SUITCIRCUITHEATEXCHANGERHEATING via CDRSUITISOLVALVE
         VALVE OUT  0 1.5								# to SUITCIRCUIT
 		VALVE OUT2  0 0.001								# suit disconnected (unused right now)
 	</TANK>
 	
-	<TANK> LMPSUIT <0.5 0.5 0.0> 75.0 0.0001
+	<TANK> LMPSUIT <0.5 0.5 0.0> 30.0 0.001
 		CHM    0 0.083125976 0.083042850 35.42964094 # Calculated for Suit Volume at 0.01PSI and 0F
 		#CHM    0 27.413288737 27.385875449 13463.26356 # Calculated for Suit Volume at 3.8PSI and 70F
         VALVE IN   1 1.0								# from SUITCIRCUITHEATEXCHANGERHEATING via CDRSUITISOLVALVE
         VALVE OUT  0 1.5								# to SUITCIRCUIT
 		VALVE OUT2  0 0.001								# suit disconnected (unused right now)
 	</TANK>
 	
-	<TANK> SUITCIRCUIT <0.5 0.5 -0.5> 30.0 0.0001
+	<TANK> SUITCIRCUIT <0.5 0.5 -0.5> 30.0 0.001
 		CHM    0 0.02078149 0.02076071 8.857410234 # Calculated for volume at 0.01PSI and 0F
         VALVE IN   1 1.0								# from Suit Isol Valves
         VALVE OUT  1 1.0								# to SUITGASDIVERTER
 		VALVE OUT2  0 1.0								# Suit Circuit Relief to CABIN 
 	</TANK>
 	
-	<TANK> SUITGASDIVERTER <0.5 0.5 -0.5> 5.0 0.0001
+	<TANK> SUITGASDIVERTER <0.5 0.5 -0.5> 5.0 0.001
 	    CHM    0.005195374 0.005190178 2.214352559 # Calculated for volume at 0.01PSI and 0F
```

Comment:
Should this be CHM **0** 0.005195374 0.005190178 2.214352559 # Calculated for volume at 0.01PSI and 0F

Or am I reading this wrong? There's a few others like this as well

Reply:
Good catch! While this doesn't hurt anything as the system fills up from the other sources, this was a copy paste error and will be fixed!